csug/syntax.stex

```diff
@@ -327,6 +327,32 @@ For example:
 \scheme{datum->syntax-object} is identical to the Revised$^6$ Report
 \scheme{datum->syntax}.
 
+%----------------------------------------------------------------------------
+\entryheader
+\formdef{quote-syntax}{\categorysyntax}{(quote-syntax \var{datum})}
+\returns a syntax object
+\listlibraries
+\endentryheader
+
+A \scheme{quote-syntax} expression is like a \scheme{quote}
+expression except that contextual information associated with
```

Comment:
Add a comma before "except"?

Reply:
I am not a native speaker, so I followed your advice and added the comma. :) (The CSUG seems to be slightly inconsistent.)